File: ticketProject4-DistributedJava/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/LogoutRequest.java
package network.objectprotocol;


import network.dto.SellerDTO;

public class LogoutRequest implements Request {
    private SellerDTO seller;

    public LogoutRequest(SellerDTO seller) { this.seller = seller; }

    public SellerDTO getSeller() {
        return seller;
    }
}

File: ticketProject7-CrossPlatform/TS-crossp/networking/request/LogoutRequest.cs
using System;
namespace networking.protocol
{

	using SellerDTO = networking.dto.SellerDTO;

	[Serializable]
	public class LogoutRequest : Request
	{
		private SellerDTO user;

		public LogoutRequest(SellerDTO user)
		{
			this.user = user;
		}

		public virtual SellerDTO Seller
		{
			get
			{
				return user;
			}
		}
	}

}